frontend/device/android/powerd.lua

```diff
@@ -58,6 +58,11 @@ function AndroidPowerD:turnOffFrontlightHW()
         return
     end
     android.setScreenBrightness(self.fl_min)
+
+    -- Devices using OnyxWarmthController also need to turn off warmth
```

Comment:
Please remove the comment. It is not accurate. Code is enough to understand what happens and is 100% true.